migrations/20180401104913_numberMappings.js

```diff
@@ -5,6 +5,7 @@ exports.up = function(knex, Promise) {
       table.uuid('id').primary();
       table.string('number', 191).unique();
       table.string('address', 191).unique();
+      table.string('WIF', 191).unique();
```

Comment:
Restored the original file and created a new one according to your recommendations. 

The WIF should also be always unique like the address. But it is not necessary because the verification is already done with the address ?

Reply:
Cool. I don't think you'll be looking anything up by WIF in the database, so don't make it unique for now.